src/main/java/net/greatstart/model/Investment.java

```diff
@@ -10,23 +14,33 @@
 import javax.persistence.ManyToOne;
 import javax.persistence.Table;
 import java.math.BigDecimal;
-import java.time.LocalDate;
+import java.time.LocalDateTime;
 
 @Data
+@FieldDefaults(level = AccessLevel.PRIVATE)
+@AllArgsConstructor
+@NoArgsConstructor
 @Entity
 @Table(name = "investments")
 public class Investment extends AbstractModel {
+
     @Column(name = "date")
-    private LocalDate dateOfInvestment;
+    LocalDateTime dateOfInvestment;

```

Comment:
Why did you remove private? If it was done for tests, it is unacceptable solution